recipes/pcre2/all/conanfile.py

```diff
@@ -28,6 +28,7 @@ class PCRE2Conan(ConanFile):
         "with_zlib": [True, False],
         "with_bzip2": [True, False],
         "support_jit": [True, False],
+        "support_callout_fork": [True, False],
```

Comment:
- should be renamed since it's a specific feature of prcre2grep, not pcre2 library.
- it should be deleted if `build_pcre2grep` is False

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names

Maybe ?

```suggestion
        "enable_grep_callout_fork": [True, False],
```

A good name here is not easy... someone previously used "support" 🤢

Reply:
Yes...im thinking in following the other options and add only `grep_` prefix:

```
grep_support_callout_fork
```

Example:
The `PCRE2_SUPPORT_JIT` is `support_jit`
The `PCRE2GREP_SUPPORT_CALLOUT_FORK` is `grep_support_callout_fork`


What do you think?

Reply:
done